service/controller/chart/v1/resource/status/create.go

```diff
@@ -0,0 +1,66 @@
+package status
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/chart-operator/service/controller/chart/v1/key"
+)
+
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	cr, err := key.ToCustomResource(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	releaseName := key.ReleaseName(cr)
+	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("getting status for release %#q", releaseName))
+
+	releaseContent, err := r.helmClient.GetReleaseContent(ctx, releaseName)
+	if helmclient.IsReleaseNotFound(err) {
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("release %#q not found", releaseName))
```

Comment:
What about this? I think this aligns more with the fact gathering primitives described in https://github.com/giantswarm/fmt/blob/master/go/log_messages.md#ensuring-state. 

```suggestion
		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("did not get status for release %#q", releaseName))
		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("release %#q not found", releaseName))
		r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
```